keps/sig-scheduling/20190317-non-preempting-priorityclass

```diff
@@ -0,0 +1,104 @@
+---
+title: Add NonPreempting Option For PriorityClasses
+authors:
+  - "@vllry"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-scheduling
+reviewers:
+  - "k82cn"
+  - "wgliang"
+approvers:
+  - "bsalamat"
+editor: Vallery Lancey
+creation-date: 2019-03-17
+last-updated: 2019-03-17
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Allow PriorityClasses To Be Non-Preempting
```

Comment:
GitHub hasn't figured out this is a markdown file. Could you rename to `20190317-non-preempting-priorityclass.md`?